app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,83 +1,164 @@
-<template class="availability-template availability-template-not-available-in-currency">
-  <%= render 'cart_form_availability_templates', type: 'not-available-in-currency' %>
+<!-- <template class="availability-template availability-template-not-available-in-currency">
+  <%#= render 'cart_form_availability_templates', type: 'not-available-in-currency' %>
 </template>
 
 <template class="availability-template availability-template-in-stock">
-  <%= render 'cart_form_availability_templates', type: 'in-stock' %>
+  <%#= render 'cart_form_availability_templates', type: 'in-stock' %>
 </template>
 
 <template class="availability-template availability-template-backorderable">
-  <%= render 'cart_form_availability_templates', type: 'backorderable' %>
+  <%#= render 'cart_form_availability_templates', type: 'backorderable' %>
 </template>
 
 <template class="availability-template availability-template-out-of-stock">
-  <%= render 'cart_form_availability_templates', type: 'out-of-stock' %>
+  <%#= render 'cart_form_availability_templates', type: 'out-of-stock' %>
 </template>
 
-<% is_product_available_in_currency = product_available_in_currency? %>
-<% default_variant = default_variant(@variants, @product) %>
+<%# is_product_available_in_currency = product_available_in_currency? %>
+<%# default_variant = default_variant(@variants, @product) %>
 
-<%= form_for :order, html: {
+<%#= form_for :order, html: {
     id: 'add-to-cart-form',
     class: 'add-to-cart-form',
```

Comment:
verify it this class effect our design ?

Reply:
no sir it was not disturbing design